conf.yaml

```diff
@@ -523,6 +523,25 @@ contents:
                 path:   docs/shared
                 exclude_branches: [ 1.0 ]
 
+         -
+            title:      Elastic Cloud on Kubernetes (ECK)
+            prefix:     en/k8s
+            tags:       Kubernetes/Reference
+            subject:    k8s
+            current:    2.2
+            branches:   [ ]
+            index:      docs/k8s/index.asciidoc
```

Comment:
Note that this PR intends to build the quick start out of the K8s repo, but https://github.com/elastic/cloud/pull/32248 targets the Cloud repo. (https://github.com/elastic/cloud/pull/32248 needs to be replaced by a new PR that targets the K8s repo.)